src/Compilers/Core/MSBuildTaskTests/TargetTests.cs

```diff
@@ -390,6 +390,115 @@ public void GenerateEditorConfigCoreHandlesMalformedCompilerVisibleItemMetadata(
             Assert.Equal("", metaName.EvaluatedValue);
         }
 
+        [Theory]
+        [InlineData(".NETFramework", "4.5", "7.3")]
+        [InlineData(".NETFramework", "4.7.2", "7.3")]
+        [InlineData(".NETFramework", "4.8", "7.3")]
+
+        [InlineData(".NETCoreApp", "1.0", "7.3")]
+        [InlineData(".NETCoreApp", "2.0", "7.3")]
+        [InlineData(".NETCoreApp", "2.1", "7.3")]
+        [InlineData(".NETCoreApp", "3.0", "8.0")]
+        [InlineData(".NETCoreApp", "3.1", "8.0")]
+        [InlineData(".NETCoreApp", "5.0", "9.0")]
+
+        [InlineData(".NETStandard", "1.0", "7.3")]
+        [InlineData(".NETStandard", "1.5", "7.3")]
+        [InlineData(".NETStandard", "2.0", "7.3")]
+        [InlineData(".NETStandard", "2.1", "8.0")]
+
+        [InlineData("UnknownTFM", "0.0", "7.3")]
+        public void LanguageVersionGivenTargetFramework(string tfi, string tfv, string expectedVersion)
```

Comment:
>LanguageVersionGivenTargetFramework [](start = 20, length = 35)

Since this test needs to be updated when new language versions are added, please call the canary from this test, or adding some other mechanism so that we remember to update this test.

Reply:
Added a test that'll fail when the default language version changes, which should be a good enough proxy to catch this situation going forward.

---
In reply to: [443006233](https://github.com/dotnet/roslyn/pull/44911#discussion_r443006233) [](ancestors = 443006233)